src/intersection-observer.js

```diff
@@ -208,6 +209,10 @@ export class IntersectionObserver extends Observable {
    * @private
    */
   flush_() {
+    if (!this.isElementInDoc_()) {
```

Comment:
> but I assume getting this.baseElement_.win.document is expensive?

Nah, that's just a C++ prop access, so not horrible. I think this check is unnecessary. If we do keep it, do a `dev.assert(this.inDoc(), 'WTF!')` so we can fix the underlying issue.